### PR TITLE
Add register blockchain app

### PR DESCRIPTION
**Pull request by woomurf**

### CODE REVIEW

**Review by jiyouang — APPROVED** [review]
LGTM, Thanks!

**Review by yoojinko — APPROVED** [review]
LGTM, Thanks!

File: src/auth.ts [hunk 1]
```diff
@@ -105,15 +102,75 @@ export default class Auth extends AinftBase {
...
+    const { adminAddress, txHash } = await this.sendRequest(
+      HttpMethod.POST,
+      trailingUrl,
+      body
+    );
+
+    console.log(
+      `The gas fee for setting the admin account of app(${appId}) set in the nft server as the owner has been sent.`
+    );
+    await this.waitTransaction(txHash, 3);
```
Comment (ehgmsdk20) [inline src/auth.ts:139]: Nice!

File: src/auth.ts [hunk 2]
```diff
@@ -105,15 +102,75 @@ export default class Auth extends AinftBase {
...
+    };
+    const trailingUrl = `register_blockchain_app`;
+    const { adminAddress, txHash } = await this.sendRequest(
+      HttpMethod.POST,
+      trailingUrl,
+      body
+    );
+
+    console.log(
+      `The gas fee for setting the admin account of app(${appId}) set in the nft server as the owner has been sent.`
```
Comment (ehgmsdk20) [inline src/auth.ts:137]: how about this?
The gas fee for setting the admin account as the owner of blockchain app has been sent.
It seems to be too long and confusing

**Review by ehgmsdk20 — APPROVED** [review]
LGTM with minor comments, thanks!

File: src/auth.ts (hunk 2 above)
Comment (woomurf) [reply src/auth.ts]: Done. Thanks!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**